src/jit/lsraxarch.cpp

```diff
@@ -152,17 +151,11 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
 
         case GT_STORE_LCL_FLD:
         case GT_STORE_LCL_VAR:
-#ifdef _TARGET_X86_
-            if (tree->gtGetOp1()->OperGet() == GT_LONG)
-            {
-                tree->gtGetOp1()->SetContained();
-                info->srcCount = 2;
-            }
-            else
-#endif // _TARGET_X86_
-            {
-                info->srcCount = 1;
-            }
+            ContainCheckStoreLoc(tree->AsLclVarCommon());
+            // TODO-XArch-CQ: This should be moved to Lowering, but it widens the types, which changes the behavior
```

Comment:
Yes it was, but apparently I missed this in merging that change - thanks for catching it. (Now it's doing it both in Lowering and here, but apparently it is idempotent since there are zero diffs.)